test/e2e/privileged.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	api "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/latest"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+const (
+	privilegedPodName       = "privileged-pod"
+	privilegedContainerName = "privileged-container"
+	imageName               = "ubuntu:14.04"
```

Comment:
fair points. i will address them.

On Wed, Sep 16, 2015 at 4:49 PM, Robert Bailey <notifications@github.com>
wrote:

> In test/e2e/privileged.go
> <https://github.com/kubernetes/kubernetes/pull/13930#discussion_r39699085>
> :
>
> > +
> > +package e2e
> > +
> > +import (
> > +	. "github.com/onsi/ginkgo"
> > +	. "github.com/onsi/gomega"
> > +
> > +	api "k8s.io/kubernetes/pkg/api"
> > +	"k8s.io/kubernetes/pkg/api/latest"
> > +	client "k8s.io/kubernetes/pkg/client/unversioned"
> > +)
> > +
> > +const (
> > +	privilegedPodName       = "privileged-pod"
> > +	privilegedContainerName = "privileged-container"
> > +	imageName               = "ubuntu:14.04"
>
> Is there a smaller image that we can use so that the image pull time
> doesn't dominate the time it takes to run this test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13930/files#r39699085>.
>